elm.json

```diff
@@ -1,18 +1,16 @@
 {
-    "type": "package",
-    "name": "stephenreddek/elm-time-picker",
-    "summary": "An elm implementation of a time picker",
-    "license": "BSD-3-Clause",
-    "version": "1.0.3",
-    "exposed-modules": [
-        "TimePicker"
-    ],
-    "elm-version": "0.19.0 <= v < 0.20.0",
-    "dependencies": {
-        "elm/core": "1.0.0 <= v < 2.0.0",
-        "elm/html": "1.0.0 <= v < 2.0.0",
-        "elm/json": "1.0.0 <= v < 2.0.0",
-        "elm/regex": "1.0.0 <= v < 2.0.0"
-    },
-    "test-dependencies": {}
-}
\ No newline at end of file
+  "type": "package",
```

Comment:
Do you have a way of undoing these formatting changes? I don't think they are necessary.